charts/microgateway/templates/configmap.yaml

```diff
@@ -91,6 +91,9 @@ data:
         - enable: {{ $simple.mapping.denyRules.enabled }}
           log_only: {{ $simple.mapping.denyRules.logOnly }}
           level: {{ $simple.mapping.denyRules.level }}
+          {{- with $simple.mapping.denyRules.exceptions }}
```

Comment:
@jubeergon 
Shouldn't this be "{{ toYaml . | nindent 12 }}"

At least it is configured this way in deployment.yaml. I suggest to use only one way to achieve such things.

Reply:
Then we would have to change the apps: as well. 
https://github.com/ergon/airlock-helm-charts/blob/9b2e1de4782156ba7cf27563ace35b14b9e02349/charts/microgateway/templates/configmap.yaml#L77

I copied it from there. 
But yes agree consistent would be better.